lib/pages/domain-my-own-page.js

```diff
@@ -2,10 +2,19 @@
 
 import { By } from 'selenium-webdriver';
 
+import * as driverHelper from '../driver-helper';
 import AsyncBaseContainer from '../async-base-container';
 
 export default class MyOwnDomainPage extends AsyncBaseContainer {
 	constructor( driver ) {
-		super( driver, By.css( '.transfer-domain-step__domain-description' ) );
+		super( driver, By.css( '.use-your-domain-step__content' ) );
+	}
+
+	async selectBuyDomainMapping() {
+		return await driverHelper.clickWhenClickable(
+			this.driver,
+			By.css( '.use-your-domain-step__option-button:not(.is-primary)' ),
+			this.explicitWaitMS
```

Comment:
Minor -` this.explicitWaitMS` isn't needed as it's the default

Reply:
Ah. Ok. Good to know. Thanks!